pkg/usagetracker/tracker.go

```diff
@@ -21,16 +20,19 @@ import (
 )
 
 type Config struct {
+	Enabled       bool                `yaml:"enabled"`
 	InstanceRing  InstanceRingConfig  `yaml:"ring"`
 	PartitionRing PartitionRingConfig `yaml:"partition_ring"`
 
 	IdleTimeout time.Duration `yaml:"idle_timeout"`
 }
 
 func (c *Config) RegisterFlags(f *flag.FlagSet, logger log.Logger) {
+	f.BoolVar(&c.Enabled, "usage-tracker.enabled", false, "True to enable the usage-tracker.")
+
 	c.InstanceRing.RegisterFlags(f, logger)
 	c.PartitionRing.RegisterFlags(f)
-	f.DurationVar(&c.IdleTimeout, "usage-tracker.idle-timeout", c.IdleTimeout, "The time after which series are considered idle and not active anymore. Must be greater than 0 and less than 1 hour.")
+	f.DurationVar(&c.IdleTimeout, "usage-tracker.idle-timeout", 20*time.Minute, "The time after which series are considered idle and not active anymore. Must be greater than 0 and less than 1 hour.")
```

Comment:
@colega I guess this was a typo we introduced in the previous PR. I've set a default of 20m.

Reply:
Thanks!